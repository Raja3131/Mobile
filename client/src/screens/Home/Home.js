import {useContext, useEffect} from 'react';
import {View, Text, Pressable} from 'react-native';
import {GlobalContext} from '../../context/Provider';

export const Home = () => {
  const {
    authDispatch,
    authState: {error, loading, isLoggedIn, data},
  } = useContext(GlobalContext);
  console.log(data);
  const onLogOut = () => {
    logoutUser()(authDispatch);

  };
//   useEffect(()=>{
//     var result = data.map(dat => ({ value: dat.id, text: dat.firstName }));
//     console.log(result)
//   },[])
  return (
    <>
      <View
        style={{
          justifyContent: 'center',
          alignItems: 'center',
        }}>
       {
      
       }
       <Text>Hi {data['user'].firstName.toUpperCase()}</Text>
      </View>
    </>
  );
};
