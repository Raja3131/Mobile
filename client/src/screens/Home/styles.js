import {StyleSheet} from 'react-native';
import appColors from '../../styles/appColors';

export default StyleSheet.create({
  homeContainer: {
    flex: 1,
   
  },
  Headers: {
    flex:1,
    justifyContent: 'center',
    alignItems: 'center',
    backgroundColor: appColors.Blue,
    shadowColor: '#333333',
    shadowOffset: {width: 10, height: 10},
    shadowRadius: 2,
    shadowOpacity: 0.4,
    padding: 20,
    borderRadius: 8,
    margin: 5,
    alignItems: 'center',
    textShadowRadius: 10,
    elevation: 10,
  },
  appointmentContainer:{
    flex:3,
  },
  iconContainer: {
    flex: 1,
    justifyContent: 'flex-end',
    flexDirection: "row",
  },
  plusIconButton:{
    marginRight: 20,
  }
});
