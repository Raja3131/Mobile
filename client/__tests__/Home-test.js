import React from 'react';
import renderer from 'react-test-renderer';
import { Home } from './../src/screens/Home/Home';

test('renders correctly', () => {
  const tree = renderer.create(<Home />).toJSON();
  expect(tree).toMatchSnapshot();
});