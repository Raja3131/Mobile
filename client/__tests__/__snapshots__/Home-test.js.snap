// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "justifyContent": "center",
    }
  }
>
  <View
    style={
      Object {
        "alignItems": "center",
        "backgroundColor": "#0190db",
        "borderRadius": 8,
        "elevation": 10,
        "margin": 5,
        "padding": 20,
        "shadowColor": "#333333",
        "shadowOffset": Object {
          "height": 10,
          "width": 10,
        },
        "shadowOpacity": 0.4,
        "shadowRadius": 2,
        "textShadowRadius": 10,
      }
    }
  >
    <Text
      style={
        Array [
          Object {
            "fontSize": 22,
            "fontWeight": "500",
            "padding": 10,
            "textAlign": "center",
          },
          Object {
            "color": "black",
          },
        ]
      }
    >
      Welcome To Heartly
    </Text>
  </View>
</View>
`;
